docs/concepts/configuration/assign-pod-node.md

```diff
@@ -206,6 +206,94 @@ If defined but empty, it means "all namespaces."
 All `matchExpressions` associated with `requiredDuringSchedulingIgnoredDuringExecution` affinity and anti-affinity
 must be satisfied for the pod to schedule onto a node.
 
+#### More Practical Use-cases 
+
+Interpod Affinity and AnitAffinity can be even more useful when they are used with higher
+level collections such as ReplicaSets, Statefulsets, Deployments, etc.  One can easily configure that a set of workloads should
+be co-located in the same defined topology, eg., the same node. 
+
+##### Always co-located in the same node
+
+In a three node cluster, a web application has in-memory cache such as redis, we want the web-servers to be co-located with the cache as much as possible. 
+Here is the yaml snippet of a simple redis deployment with two replicas and selector label `app=store`
+
+```yaml
+apiVersion: apps/v1beta1 # for versions before 1.6.0 use extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redis-cache
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: store
+    spec:
```

Comment:
Hmmm, i considered this also but we were trying to emphasis `podAffinity` in this example we are separately linking the user to a zookeeper example for `podAnitAffinity`, i was thinking we should keep this example simple and clear. We may very well add a sentence saying the best practice would be to add anti-affinity for `app=store` workload. What do you think?